packages/hardhat-waffle/src/type-extensions.ts

```diff
@@ -10,21 +10,23 @@ import type { ContractJSON } from "ethereum-waffle/dist/esm/ContractJSON";
 import type { Contract, providers, Signer } from "ethers";
 import "hardhat/types/runtime";
 
+interface HardhatWaffle {
```

Comment:
I think this should be placed in another file. Probably it's own file.

Reply:
Done- let me know if this is what you had in mind

Reply:
For consistency with how this is done in the rest of our packages, I would move it to a `src/types.ts` file instead of `src/HardhatWaffle.ts`

Reply:
Oh, and I'm pretty sure you are going to get a linter error about import groups. You can run `yarn lint` inside the package directory to check that.

Reply:
Renamed to `types.ts`. Also ran linting check, and got no errors.